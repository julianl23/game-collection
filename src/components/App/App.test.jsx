import React from "react";
import TestRenderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
// import App from "./App";

describe.skip("App", () => {
  // Throws an error about symbol is not a function.
  // Related to styled-components globals
  it.skip("renders correctly", () => {
    const instance = TestRenderer
      .create
      // <BrowserRouter>
      //   {/* <App /> */}
      // </BrowserRouter>
      ();
    expect(instance.toJSON()).toMatchSnapshot();
  });
});
