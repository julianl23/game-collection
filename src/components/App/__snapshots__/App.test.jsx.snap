// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
Array [
  .c0 {
  height: 60px;
  background: #d8d8d8;
  border: 1px solid #999;
  border-left: 0;
  border-right: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 25px 0 25px;
}

.c2 {
  font-size: 1.5em;
  margin: 0;
}

.c1 {
  margin-right: 20px;
  fill: #fff;
}

<header
    className="c0"
  >
    <svg
      className="c1"
      style={
        Object {
          "fill": "#fff",
          "height": "20px",
          "width": "25px",
        }
      }
      viewBox="0 0 640 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M480 96H160C71.6 96 0 167.6 0 256s71.6 160 160 160c44.8 0 85.2-18.4 114.2-48h91.5c29 29.6 69.5 48 114.2 48 88.4 0 160-71.6 160-160S568.4 96 480 96zM256 276c0 6.6-5.4 12-12 12h-52v52c0 6.6-5.4 12-12 12h-40c-6.6 0-12-5.4-12-12v-52H76c-6.6 0-12-5.4-12-12v-40c0-6.6 5.4-12 12-12h52v-52c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v52h52c6.6 0 12 5.4 12 12v40zm184 68c-26.5 0-48-21.5-48-48s21.5-48 48-48 48 21.5 48 48-21.5 48-48 48zm80-80c-26.5 0-48-21.5-48-48s21.5-48 48-48 48 21.5 48 48-21.5 48-48 48z"
      />
    </svg>
    <h1
      className="c2"
    >
      Game Collection
    </h1>
  </header>,
  .c0 {
  display: grid;
  grid-template-columns: repeat(4);
  grid-gap: 16px;
  grid-auto-rows: minmax(100px,auto);
  min-height: 90vh;
  padding: 25px;
}

.c1 {
  grid-column: 1 / 4;
  grid-row: 1;
}

<section
    className="c0"
  >
    <section
      className="c1"
    >
      Whoa! Main Content
    </section>
  </section>,
  .c0 {
  background: #d8d8d8;
  border: 1px solid #999;
  border-left: 0;
  border-right: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 10px 25px;
}

<footer
    className="c0"
  >
    Â© 
    2018
     Julian Lord
  </footer>,
]
`;
